### ISSUE TITLE
Don't make me install graphviz 

### ISSUE THREAD

**Issue by steren** [ahmetb/cloudrun-iamviz#4]
I want to `docker run` this

**Comment by ahmetb** [comment]
Manufacturing creds and mounting them to a container be harder than installing the binary.

**Comment by steren** [comment]
What about deploying this as a cloud run service that has the default service account in my project using the Button?

**Comment by ahmetb** [comment]
Yeah that could work, too. But then most likely you'd need some sort of authz to prevent service names accidentally leaking to public. The easiest way would be to run it as it is today –or inside a local `docker run` with ACCESS_TOKEN passed from outside as an env var.
